src/content/updates/24-05-16-fujocoded-llc-down-to-business/index.md

```diff
@@ -0,0 +1,78 @@
+
+### Recent Progress
+
+Though we did take a few weeks to focus on launching FujoCoded, LLC, we’re back at it with more progress on the Fujoshi Guide to Web Development:
+
+- **Vol. 1, Issue 1: Git is being prepared for early release** to our KickStarter backers so that we can get feedback from all of _YOU_ while we are working on the final art! Instead of final art, the early release version will have Ms Boba’s hilarious sketches, which should honestly count as bonus content 👍
+- **Vol. 1, Issue 2: GitHub** is (almost) entirely written! Ms Boba put her head down and powered through the Guide’s main scenario, leaving just the “extra credits” sections to be finalized before our first beta round (soon to be scheduled). Excited to see how this issue’s exercise resonates with our readers!
+- **We have started work on the FujoGuide wiki**, which will house any and all online reference content mentioned in the text of the guide itself.
+
+### What’s Next
+
+- **The text-and-sketches early-release version of Vol. 1, Issue 1: Git** **will be sent out as a PDF to KickStarter backers** via BackerKit’s reward delivery tool. We don’t have an exact date, but our _wide-eyed idealist dream_ is to get it to you around the end of June 🤞
+- **Vol. 1, Issue 2: GitHub** **will be handed over to a volunteer for high-level “alpha” feedback** so we can make adjustments prior to the first round of beta reading. We’ll also start planning the schedule of our first beta feedback round.
+- Next Thursday, Ms Boba will be **livestreaming a look at how we’re using Astro to build FujoGuide**’s print layout and web version at the same time! Since our streams will focus on the work we must do for fulfillment, expect FujoGuide to pop up from time to time. Check out the schedule below, and come say hi!
+
+## Hang Out with FujoCoded on Twitch 💬✨
+
+For the rest of May and through all of June, M**s Boba will be streaming only once a week** while she focuses on fulfillment for the recently concluded BackerKit campaign to launch FujoCoded LLC, and on wrapping up the writing for FujoGuide Volume 0.
+
+There may be an extra bonus stream here or there, so keep an eye on our social media! You can check [Ms Boba’s “streaming schedule” page here.](https://essentialrandomness.com/streams)
+
+Currently, the schedule is as follows:
+
+<figure><img src="https://assets.buttondown.email/images/1452996e-2d28-4c2f-ad9f-752a05d79149.png?w=960&amp;fit=max" alt="May  Thursday, May 16 @ 3pm PDT: Let’s finish the $ponsors $hrine!  Thursday, May 23 @ 3pm PDT: [Tentative] Building FujoGuide with Astro  Thursday, May 30 @ 3pm PDT: Something BobaBoard  June  Thursday, June 6 @ 3pm PDT: TBD  Thursday, June 13 @ 3pm PDT: TBD  Thursday, June 20 @ 3pm PDT: Something BobaBoard  Thursday, June 27 @ 3pm PDT: TBD" draggable="false" contenteditable="false"><figcaption>A lot of TBD, which are really “whatever we need to get done that’d interesting/educative for people to watch us work on”</figcaption></figure>
```

Comment:
Likewise we need a better styling for figcaption. Don't worry too much about it, just make it smaller and centered. Also add some margins to the figure so it creates a visible block.